server/views/routes/users.js

```diff
@@ -0,0 +1,16 @@
+const router = require('express').Router();
+const registerUser = require('../../controllers/user/register');
+const responseHandler = require('../../utils/response-handler');
+
+router.post('/register', async (req, res) => {
+  try {
+    const { email } = req.body;
+
+    await registerUser(email.toLowerCase());
```

Comment:
What happens after the user is registered (new user entity added to the DB)? Perhaps the next feature/PR you work on could be to add authentication.. i.e creating an access token and saving a cookie with that token to the client.

Reply:
Yup, this is the next feature that I am planning to work on. The authentication/login component.